lib/generate_types.rb

```diff
@@ -20,7 +20,7 @@ def self.generate_facility_location
 
     def self.generate_code_field
       # TODO: use code with all parameters, possibly randomize parameter values and optional information
-      QDM::Code.new('1234', 'SNOMED-CT')
+      QDM::Code.new('1234', 'SNOMEDCT')
```

Comment:
The cqm-parsers `CODE_SYSTEMS` hash has the name of this code system as `SNOMED-CT`. Do we want this to be different than that?

Reply:
Well...more important question...is "CodeSystemHelper" now dead code in cqm-parsers now that VSACValueSetLoader just uses the valueset name from VSAC.

The CodeSystemHelper is used in the method "retrieve_code_system_for_model" in data_criteria.rb (which is called by the "to_model" method).  I'm not versed enough in cqm-parsers to know if this is also just old/dead code.

Reply:
That makes sense. 

Given that, does it make sense to remove `SNOMED-CT` everywhere it exists in this repo? It's also present in some of the fixtures (patient/value set), and in the RSpec tests.